app/Main.hs~

```diff
@@ -0,0 +1,8 @@
+module Main where
```

Comment:
[pebble] duplicate file?

Reply:
I have to recursively apply `*~` to my `.gitignore` to keep emacs' backup files out of my commits. Evidently `*~` only worked at the top level.

Reply:
Lol I just realized that I forgot to escape the tilde.